src/project.rs

```diff
@@ -229,6 +242,16 @@ impl FuzzProject {
         Ok(cmd)
     }
 
+    fn with_coverage_output_extension(filename: &str) -> Result<String> {
+        let file_with_ext = Path::new(filename).with_extension("profraw");
+        match file_with_ext.to_str() {
+            Some(name) => Ok(String::from(name)),
+            None => {
+                bail!("File name for generated coverage output must contain only UTF-8 characters.")
```

Comment:
You can pass `Path`/`PathBuf` to `env` directly, avoiding the conversion back to `String` and thus this failure mode.